src/tools/check_versions/src/cli.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::path::PathBuf;
+
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+/// Version checking utility to identify which project components may need upgrading to the latest
+/// version. Currently only some types of components can be automatically checked for updates - this is not an
+/// exhaustive list.
+pub struct Args {
+    #[arg(short, long, required = true)]
+    /// The versions yaml file listing components to check for updates.
+    /// Intended to be used with the versions.yaml file in the root of the kata-containers project.
+    pub versions_file: PathBuf,
```

Comment:
It might be a nice optimisation to make this optional so that the user can simply run:

```bash
$ check_versions ../../../versions.yaml
```

```suggestion
    pub versions_file: Option<PathBuf>,
```